Makefile

```diff
@@ -9,14 +9,14 @@ all: list
 
 install:
 	./tools/generate_key_pair.sh
-	pip install --upgrade -r requirements.dev.txt
+	pip install --upgrade -e .[develop]
 	export PRIVATE_KEY=`cat private.pem`
```

Comment:
Why not do all this in Travis? Why should we generate public private keys in the makefile?